web/src/components/TestSpecForm/TestSpecForm.tsx

```diff
@@ -110,26 +110,24 @@ const TestSpecForm = ({
           <SelectorInput form={form} testId={testId} runId={runId} onValidSelector={setIsValid} />
 
           <S.SuggestionsContainer>
-            {!isEditing && (
-              <SelectorSuggestions
-                onClick={query => {
-                  onClickPrevSelector(form.getFieldValue('selector'));
-                  onClearSelectorSuggestions();
-                  form.setFieldsValue({
-                    selector: query,
-                  });
-                }}
-                onClickPrevSelector={query => {
-                  onClickPrevSelector('');
-                  onClearSelectorSuggestions();
-                  form.setFieldsValue({
-                    selector: query,
-                  });
-                }}
-                prevSelector={prevSelector}
-                selectorSuggestions={selectorSuggestions}
-              />
-            )}
+            <SelectorSuggestions
+              onClick={query => {
```

Comment:
Removing the isEditing check as per this ticket https://github.com/kubeshop/tracetest/issues/2008